tasks/install_prometheus.yml

```diff
@@ -14,7 +14,7 @@
 - name: Copy config file
   become: yes
   template:
-        src: prometheus.yml.j2
+        src: "{{ prometheus_config_template }}"
```

Comment:
This is the same change as in #4 isn't it? Until I decide on what to do with this PR, the changes should be exclusives, so I'd revert 28e4258 and f87398b